src/doc/unstable-book/src/language-features/c-unwind.md

```diff
@@ -0,0 +1,14 @@
+# `c_unwind`
+
+The tracking issue for this feature is: [#74990]
+
+[#74990]: https://github.com/rust-lang/rust/issues/74990
+
+------------------------
+
+Introduces a new ABI string, "C-unwind", to enable unwinding from other
+languages (such as C++) into Rust frames and from Rust into other languages.
```

Comment:
This should mention the other variants as well (`"system-unwind"`, etc).

Reply:
This has been fixed in fc35079, thank you for pointing that out! :slightly_smiling_face: